CHANGES.rst

```diff
@@ -4,7 +4,10 @@ astropy-helpers Changelog
 3.2 (unreleased)
 ----------------
 
-
+- Remove functionality to adjust compilers if a broken compiler is detected.
+  This is not useful anymore as only a single compiler was previously patched
+  (now unlikely to be used) and this was only to fix a compilation issue in the
```

Comment:
Is there an issue about the *original* problem?  If so it might be good to link to it *somewhere*, and this seems the logical place.

Reply:
Looks like the original issue was https://github.com/astropy/astropy/issues/31 - I'll open a PR for a known issue.

Reply:
Done - it's in the README now.